tests/test_dev_server.py

```diff
@@ -1,6 +1,8 @@
 # coding=utf-8
 import io
+import json
 import socket
+import urllib
```

Comment:
I think you should import `urllib.request`, otherwise, PyCharm has problems finding the correct code (most likely `mypy` will have the same problems).